src/main/java/io/ebeaninternal/server/core/DefaultBeanLoader.java

```diff
@@ -288,7 +288,7 @@ private void refreshBeanInternal(EntityBean bean, SpiQuery.Mode mode, int embedd
     }
 
     if (embeddedOwnerIndex > -1) {
-      query.select("id," + ebi.getProperty(embeddedOwnerIndex));
```

Comment:
Maybe this was the select that triggers the NPE mentioned below... I really should write a test that addresses this issue...